test/CMakeLists.txt

```diff
@@ -1983,31 +2002,33 @@ endif()
 # Land product converters
 #==============================================================================
 
-if( iodaconv_pygrib_ENABLED )
-  ecbuild_add_test( TARGET  test_${PROJECT_NAME}_ims_scf
-                    TYPE    SCRIPT
-                    ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
-                    COMMAND bash
-                    ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
-                            netcdf
-                            "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/ims_scf2ioda.py
-                            -i testinput/imssnow_24km.grib2
-                            -o testrun/imssnow_scf.nc
-                            -m maskout"
-                            imssnow_scf.nc ${IODA_CONV_COMP_TOL_ZERO})
-
-  ecbuild_add_test( TARGET  test_${PROJECT_NAME}_afwa_snod
-                    TYPE    SCRIPT
-                    ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
-                    COMMAND bash
-                    ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
-                            netcdf
-                            "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/afwa_snod2ioda.py
-                            -i testinput/afwa_snod_24km.grib
-                            -o testrun/afwa_snod.nc
-                            -m maskout"
-                            afwa_snod.nc ${IODA_CONV_COMP_TOL_ZERO})
-endif()
+# The following two tests have never been enabled and the python codes were not updated
+# for JEDI data naming conventions.  So these are now disabled until updates are moade.
+#if( iodaconv_pygrib_ENABLED )
+#  ecbuild_add_test( TARGET  test_${PROJECT_NAME}_ims_scf
```

Comment:
pinging @YoulongXia-NOAA and @jiaruidong2017 , are these needed or can they be removed? I know we are able to use different formats besides the GRIB2 files now.

Reply:
@CoryMartin-NOAA. I recalled that I removed these grib-based ioda converters and tests as the python did not support pygrib well as we discussed before.

Reply:
@CoryMartin-NOAA, this is what I did before (#883)

Reply:
@YoulongXia-NOAA   Just to be clear.  If there is a desire to have these data in JEDI, I can offer to transition the python codes to follow the naming conventions and these 2 codes could be enabled in approximately two months.  Furthermore, if we identify a useful package that is lacking in our software stack, then it is better to speak up early to request its addition.

Reply:
I think these datasets are used with another input format, as I know we are using IMS snow cover data for JEDI snow DA.

Reply:
@gthompsnJCSDA, the Land DA group decided to no use afwa snow data for land DA, and they decided to not use ims grib file either. So that I removed them. Thank you.